PyPDF2/generic.py

```diff
@@ -1163,9 +1163,25 @@ def get_data(self):
             self.decoded_self = decoded
             return decoded._data
 
+    def getData(self):
```

Comment:
The other deprecated functions in this file do not have the `# pragma: no cover` so I opted to not include it here like I did in #1139 (where all deprecated functions do have the pragma). Wasn't sure if should include it (and so that would probably get codecov to pass) or not include it.